extension_templates/classification.py

```diff
@@ -273,6 +274,14 @@ def get_test_params(cls, parameter_set="default"):
         # A parameter dictionary must be returned *for all values* of parameter_set,
         #   i.e., "parameter_set not available" errors should never be raised.
         #
+        # A good parameter set should primarily satisfy two criteria,
+        #   1. Chosen set of parameters should reduce the testing time.
```

Comment:
"reduce" is vague, since it refers to a baseline point which is unclear.

Better: "keep testing time low"

"low" is also vague, so try to say sth about the order, e.g., less than a second for the entire test suite?

Reply:
How about "lower testing time than the default set"? I think it'll be better than giving a value because otherwise, it sounds like this should actively be enforced (using something like `timeit`).

A lower-than-default time should suffice.

Reply:
"lower" can still be hours, I think we should tell people what they should approximatel be aiming for (but also imply it's approximate and order of magnitude)